### PR TITLE
core: Api Integration First Card

### PR DESCRIPTION
**Pull request by sawood14012**
Signed off by msawood@redhat.com
 
ADDS API for the first card

### CODE REVIEW

File: src/app/components/shared-components/addons-primary/drawerNew.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+const FeedbackButtons = () => {
+  return (
+    <div>
+      <Button variant="link" icon={<FontAwesomeIcon icon={faThumbsUp} />} />
+      <Button variant="link" icon={<FontAwesomeIcon icon={faThumbsDown} />} />
+    </div>
+  )
+}
+
+const DrawerFC = () => {
```
Comment (rafiu007) [inline src/app/components/shared-components/addons-primary/drawerNew.tsx:53]: What happened to the drawerStack that i had created?Why is that not reused?I feel that should have been reused as the drawer development for that is also done.

Comment (sawood14012) [reply src/app/components/shared-components/addons-primary/drawerNew.tsx:53]: i didnt want to mess up your code so i created a new file you can take this code and modify yours and delete this file think of it as a temp code

Comment (rafiu007) [reply src/app/components/shared-components/addons-primary/drawerNew.tsx:53]: Sure.

File: src/app/components/shared-components/addons-primary/drawerNew.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      </DrawerHead>
+    </DrawerPanelContent>
+  )
+  return (
+    <Drawer isExpanded={drawerSta}>
+      <DrawerContent panelContent={panelc}>
+        <DrawerContentBody>
+          <div>
+            <Table
+              aria-label="Sortable Table"
```
Comment (rafiu007) [inline src/app/components/shared-components/addons-primary/drawerNew.tsx:88]: In this approach how are we planning to pass api-data from drawer to rows?

Comment (sawood14012) [reply src/app/components/shared-components/addons-primary/drawerNew.tsx:88]: you have to get the data from the APi then create a util function which generates a list likewise how i have did then you just pass on this to the table as rows

**Review by rafiu007 — APPROVED** [review]
LGTM

**Pull request merged**